include/modem/location.h

```diff
@@ -322,6 +337,25 @@ const char *location_method_str(enum location_method method);
  */
 int location_agps_data_process(const char *buf, size_t buf_len);
 
+/**
+ * @brief Feed in P-GPS data to be processed by library.
+ *
+ * @details If Location library is not receiving P-GPS assistance data directly from nRF Cloud,
+ * it throws event LOCATION_EVT_GNSS_PREDICTION_REQUEST when assistance is needed. Once application
```

Comment:
```suggestion
 * it triggers the LOCATION_EVT_GNSS_PREDICTION_REQUEST event when assistance is needed. Once application
```

Reply:
Thank you, sir. I implemented all your changes as proposed.